README.md

```diff
@@ -1,23 +1,118 @@
 # clp-bench
-clp-bench is a tool for benchmarking [CLP] as well as other log management tools. The tool itself is
-a Python package, and we also provide a [web interface][ui] for viewing results.
 
-The methodology for the benchmarks is described [here](docs/methodology.md).
+**clp-bench** is a benchmarking tool designed for [CLP] and other log management systems. It
+functions as a Python package and includes a [web interface][ui] for displaying benchmark results.
+
+For a detailed description of the benchmarking methodology, see
+[this document](docs/methodology.md).
 
 ## Requirements
 
-* Docker
-* Python v3.10 or higher
+- Docker
+- Python v3.10 or higher
 
-# Set up
+# Setup
 
 ```shell
 python3 -m venv venv
 . venv/bin/activate
 pip install -e .
 ```
 
-You can use `clp-bench --help` to see usage instructions.
+To view usage instructions, run `clp-bench --help`.
+
+# Contributing
+
+🚧 This section is under construction.
+
+We encourage contributions that add benchmark results for various tools to support broader community
+development.
+
+## Adding new results
+
+To benchmark a new system, duplicate one of the directories in [assets] and update the following
```

Comment:
Instead of asking the user to duplicate one of the dirs for an existing tool, can we create a template dir and ask them to duplicate that and fill in the necessary files?